aws_sagemaker_studio/getting_started/xgboost_customer_churn_studio.ipynb

```diff
@@ -431,9 +431,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Trial 2 - XGBoost in Framework mode\n",
+    "#### Trial 2 - XGBoost in framework mode\n",
     "\n",
-    "For the next trial, we'll train a similar model, but using XGBoost in framework mode. This way of using XGBoost should be to familiar to users who have worked with the open source XGBoost. Using XGBoost as a framework provides more flexibility than using it as a built-in algorithm as it enables more advanced scenarios that allow pre-processing and post-processing scripts to be incorporated into your training script. Specifically, we will be able to specify a list of rules that we want the SageMaker Debugger to evaluate our training process against."
+    "For the next trial, we'll train a similar model, but we'll use XGBoost in framework mode. If you've worked with the open source XGBoost framework, this way of using XGBoost should be  familiar to you. Using XGBoost as a framework provides more flexibility than using it as a built-in algorithm because it enables more advanced scenarios that allow you to incorporate preprocessing and post-processing scripts into your training script. Specifically, we'll be able to specify a list of rules that we want Debugger to evaluate our training process against."
```

Comment:
Minor:

...we'll train a similar model, but use XGBoost in framework mode.

Do you mean script mode? We should use common terminology.